processor/labelsprocessor/testdata/config.yaml

```diff
@@ -0,0 +1,29 @@
+receivers:
+  examplereceiver:
+
+processors:
+  labels_processor:
+    labels:
+      - key: label1
+        value: value1
+      - key: label2
+        value: value2
+  labels_processor/env_vars:
+    labels:
+      - key: label1
+        value: $VALUE_1
+      - key: label2
+        value: $VALUE_2
```

Comment:
I'm not too sure but I think the convention is to wrap environment variables in yaml files with `${}` so like it would be `${VALUE_1}`

Reply:
the convention is what I put in this file. Environment variable parsing is actually already done in another part of the collector. By the time the configuration data reaches the processor code I made, environment variables have already been dealt with

Reply:
Look at the bottom of this link: https://opentelemetry.io/docs/collector/configuration/